xds/xds.go

```diff
@@ -38,6 +38,7 @@ import (
 
 	_ "google.golang.org/grpc/credentials/tls/certprovider/pemfile"         // Register the file watcher certificate provider plugin.
 	_ "google.golang.org/grpc/xds/internal/balancer"                        // Register the balancers.
+	_ "google.golang.org/grpc/xds/internal/clusterspecifier/rls"            // Register the RLS cluster specifier plugin.
```

Comment:
Include a note that this does not also register the LB policy itself?

Reply:
Done.